system/coverage/browser/CodeBrowser.cfc

```diff
@@ -91,7 +91,8 @@ component accessors=true {
 				var fileContents            = fileRead( fileData.filePath );
 
 				var levelsFromRoot = fileData.relativeFilePath.listLen( "/" );
-				var relPathToRoot = RepeatString('../', levelsFromRoot - 1 );
+				var relPathToRoot = RepeatString("../", levelsFromRoot - 1 );
+				var brush = right(fileData.relativeFilePath,4) == ".cfm" ? "coldfusion" : "javascript";
```

Comment:
Please always use a spacing before and after parentheses, and commas.

`right( fileData.relativeFilePath, 4 )`